.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
 
 jobs:
   integration_tests:
-    runs-on: ubuntu-latest
```

Comment:
ideally we'd split these .yaml changes into a separete PR

Reply:
Ideally, but it gets the build working so low impact. Agreed though in general if it wasn't blocking this build.